Testing/functional/e2e/cypress/integration/services/webClient/dependent.js

```diff
@@ -83,16 +83,6 @@ describe('WebClient Dependent Service', () => {
         })
     })
 
-    it('Verify Put Dependent Not Found', () => {
```

Comment:
This is the only place we have the test for 404. The issue is we get 404 only in local as we allow HTTP. In DEV, as HTTP is not allowed, the test fails as it returns 405 instead. Here is a related article for 405 error: 
![image](https://user-images.githubusercontent.com/48332333/106004750-61e20600-6068-11eb-9c8a-9add009ccf6a.png)

Reply:
A 401 and a 405 isn't unauthorized so I think the test needs to be fixed to actually verify unauthorized.  We should never be hitting HTTP endpoints using our tests that would be another defect in the test.

Reply:
We already have the 401 test which is a bit above on the same file
`it('Verify Get Dependents Unauthorized', () => {
        cy.request({ 
            url: `${BASEURL}${HDID}/Dependent`,
            followRedirect: false,
            failOnStatusCode: false
        })
        .should((response) => { expect(response.status).to.eq(401) })
    })    `